hieradata/common.yaml

```diff
@@ -1706,7 +1706,7 @@ mysql::client::package_ensure: 'present'
 nginx::package::version: '1.4.6-1ubuntu3.8'
 
 nodejs::repo::apt_mirror_hostname: "%{hiera('apt_mirror_hostname')}"
-nodejs::version: '6.11.2-1nodesource1~%{::lsbdistcodename}1'
+nodejs::version: '6.14.3-1nodesource1~%{::lsbdistcodename}1'
```

Comment:
This doesn't have the `~%{::lsbdistcodename}1` on it anymore

in apt it's:
`Version: 6.14.3-1nodesource1`

Reply:
Changed